assets/src/edit-story/components/checklist/checks/storyMissingTitle.js

```diff
@@ -22,18 +22,24 @@ import { useStory } from '../../../app';
 import { states, useHighlights } from '../../../app/highlights';
 import { PRIORITY_COPY } from '../../../app/prepublish/newConstants';
 import { ChecklistCard, ChecklistCardStyles } from '../../checklistCard';
+import { useRegisterCheck } from '../checkCountContext';
 
 export function storyMissingTitle(story) {
-  return typeof story.title !== 'string' || story.title?.trim() === '';
+  return (
```

Comment:
I updated this in #8035 to destructure `story` on line 34 to avoid changing the function here. I'm fine with either really, i just wanted to share that deet.

Reply:
Ah awesome! yea that's a better move